packages/sso-login/src/components/ProviderButton/StyledProviderButton.js

```diff
@@ -3,18 +3,18 @@ import {theme} from 'styled-system'
 import {StyledButton} from 'tocco-ui/src/Button'
 import {
   declareInteractionColors,
-  generateCustomColors
+  generateInteractionColors
 } from 'tocco-ui/src/utilStyles'
 
 export default styled(StyledButton)`
   && {
     display: inline-block;
     width: 100%;
     ${props => declareInteractionColors(
-    generateCustomColors(
-      props.primaryColor,
-      props.secondaryColor || theme('colors.paper')(props),
-      props.secondaryColor || theme('colors.text')(props)))}
+    generateInteractionColors(props, {
+      bg: props.primaryColor,
+      fg: props.secondaryColor || `${theme('colors.paper')(props)}, ${theme('colors.text')(props)}`
```

Comment:
dont forget to edit this line as soon as you rebased. (you removed theme dependency)